src/App.svelte

```diff
@@ -61,8 +63,16 @@
         medicareTaxes = fica[1];
 
         // Update tax info
-        totalTaxes = incomeTaxes + socialSecurityTaxes + medicareTaxes;
+        totalTaxes = incomeTaxes + socialSecurityTaxes + medicareTaxes - nonRefundableTaxCredit;
+        if (totalTaxes < 0) {
+            totalTaxes = 0;
+        }
         afterTaxIncome = grossIncome - totalTaxes;
+        totalTaxes -= refundableTaxCredit;
+        if (totalTaxes < 0) {
+            afterTaxIncome -= totalTaxes;
+            totalTaxes = 0;
+        }
```

Comment:
We can remove this entire if statement preventing totalTaxes from being negative, I think it's okay to say you have a negative tax bill

Reply:
Total taxes can now be negative. Also, the deal with the issue of having nonzero Medicare and social security taxes, I subtracted the value of the credits from income taxes. I read somewhere that tax credits usually apply to income taxes. This way, even if Medicare and SS taxes are nonzero, income taxes will be negative, and the total taxes paid will be zero.

Reply:
Oh wait that great! Can you put a link to that so I can check it out too?